docs/sad.md

```diff
@@ -67,4 +67,44 @@ Define any standards that must be followed for design, code and unit testing, pr
 Hier kan tdd mooi worden beschreven voor het gebruik binnen het project.
 }}
 
+## 6
 
+### 6.1 Non-Functional Requirements
+
+- Use of 32 bit keys.
+- A VM for verifing smart contracts.
+- Every user can have one or more account numbers.
+- Every account number is unique.
+- Authentication and integerity of transactions have to be guaranteed.
+- Validating needs to be efficient. (No proof of work)
```

Comment:
misschien vervangen door 'Transaction validation uses the Proof-of-Stake algorithm'?